src/geobr.jl

```diff
@@ -0,0 +1,288 @@
+# -----------------------------------------------------------------
+# YEAR: 2020
+# COPYRIGHT HOLDER: Institute for Appled Economic Research (Ipea)
+# -----------------------------------------------------------------
+
+module Geobr
+
+using GeoIO
+using DataDeps
+using CSV
+
+const APIVERSIONS = (v"1.7.0",)
+function downloadmetadata()
+    url = "http://www.ipea.gov.br/geobr/metadata/metadata_1.7.0_gpkg.csv"
+    return CSV.File(download(url))
+end
+
+function selectmetadata(geo; simplified=true, year=2010)
+    metadata = downloadmetadata()
+    filtered = if "simplified" in propertynames(metadata)
+        filter(row -> row.geo == geo && row.year == year && row.simplified == simplified, metadata)
+    else
+        filter(row -> row.geo == geo && row.year == year, metadata)
+    end
+    if isempty(filtered)
+        throw(ArgumentError("No metadata found for the given parameters"))
+    end
+    return filtered
+end
+
+function downloadgpkg(metadata)
+    urls = [row.download_path for row in metadata]
+    gpkgs = [GeoIO.load(download(url)) for url in urls]
+    return reduce(vcat, gpkgs)
+end
+
+function readstate(codestate, year=2010, simplified=true, verbose=false)
+    metadata = selectmetadata("state", simplified=simplified, year=year)
+    gdf = downloadgpkg(metadata)
+    if codestate == "all"
+        return gdf
+    else
+        return gdf[gdf.code_state.==codestate.||gdf.abbrev_state.==codestate, :]
+    end
+end
+
+function readmunicipality(codemuni="all", year=2010, simplified=true, verbose=false)
```

Comment:
codemuni --> code

Also, these functions seem to have a mandatory argument plus a few optional arguments. The mandatory argument must be a positional argument, whereas the optional ones we can use kwargs.